### PR TITLE
Add common opt at case-lib/opt.sh

### PR DESCRIPTION
**Pull request by Bin-QA**
1. As we have lot of same description option so move them
into the common function, so when modify/add can use it
this change is not refer the code logic

2. Change opt parse error exit code to exit 2
Current define exit code
0: pass; 1: fail;2: skip

Signed-off-by: Wu, BinX <binx.wu@intel.com>

### CODE REVIEW

**Comment by wenqingfu** [comment]
I would say the code readability drops because of your 1). why bother?  

2) should be a separate PR, don't mix changes together.

**Comment by Bin-QA** [comment]
> I would say the code readability drops because of your 1). why bother?
yes, you are right, I am not realize the readability problem  
Because I will try to add the common option for the almost script  
I will redesign the script process

> 1. should be a separate PR, don't mix changes together.
Got it.

File: case-lib/opt.sh [hunk 1]
```diff
@@ -101,7 +101,7 @@ func_opt_parse_option()
         [ ! "$idx" ] && idx="${_op_long_lst[$1]}"
         if [ "$idx" ]; then
             if [ ${OPT_PARM_lst[$idx]} -eq 1 ]; then
-                [ ! "$2" ] && echo "option: $1 missing parameter, parsing error" && exit 1
+                [ ! "$2" ] && echo "option: $1 missing parameter, parsing error" && exit 2
```
Comment (xiulipan) [inline case-lib/opt.sh]: can we use `dlog` here instead of echo

**Comment by Bin-QA** [comment]
After check with requirement, the new PR to fix exit code
```
man bash :
All builtins return an exit status of 2  to  indicate  incorrect
usage, generally invalid options or missing arguments.
```

So I will check with script to add comment of exit 2 means

**Pull request closed without merging**